macOS/LocalPackages/NewTabPage/Sources/NewTabPage/Favorites/NewTabPageFavoritesModel.swift

```diff
@@ -107,9 +107,11 @@ public final class NewTabPageFavoritesModel<FavoriteType, ActionHandler>: NSObje
     // MARK: - Actions
 
     @MainActor
-    func openFavorite(withURL url: String, target: NewTabPageDataModel.ActivityOpenAction.OpenTarget) {
+    func openFavorite(withURL url: String, target: NewTabPageDataModel.ActivityOpenAction.OpenTarget, sourceWindow: NSWindow?) {
```

Comment:
Now that we’re not using `target` here, it can be removed from the parameters here.